### PR TITLE
updates to react 0.13

### PR DESCRIPTION
**Pull request by iamdustan**
Switch to babel, remove peer dependency constraint, `npm test`, `npm run watch`, and `npm prepublish` are all hooked up. Travis yml file if you want to connect some CI.

I don’t think this is 100% there yet, fyi.

### CODE REVIEW

**Comment by iamdustan** [comment]
Updated. This is working for me now.

File: src/reactComponents.js [hunk 1]
```diff
@@ -1,22 +1,23 @@
...
     var level = this.props.component.level;
-    if (level && level > 0 && level < 7) {
-      html = React.DOM['h' + level](null, values);
-    } else {
-      html = React.DOM.div(null, "ERROR incorrect level");
-    }
-    return (
-      html
+    invariant(
+      between(level, 0, 6),
```
Comment (tcoopman) [inline src/reactComponents.js]: I think this should be `between(level, 0, 7)`

**Comment by tcoopman** [comment]
Looks good, if you fix the little mistake, I'll merge it.

Thank you for the nice work!!

**Comment by iamdustan** [comment]
Done! Also to note, I haven’t updated all of the examples so the other two likely don’t work at this point...

**Comment by tcoopman** [comment]
Ok, no problem. That can be done later.

**Pull request merged**